setup.cfg

```diff
@@ -67,7 +67,7 @@ docs =
     myst-nb
 
 [bumpver]
-current_version = "v2.2.0a0"
+current_version = "v2.2.0a1"
 version_pattern = "vMAJOR.MINOR.PATCH[PYTAGNUM]"
 commit_message = "Bump version {old_version} -> {new_version}"
 commit = True
```

Comment:
If the tag from the fork is not sync to the repo, then it is meaningless to tag to the forked repo, we need to remove `tag = True`.